bgp_adjacencies/BGP_Neighbors_Established.py

```diff
@@ -87,7 +87,9 @@ def check_bgp(self):
         for device, bgp in self.all_bgp_sessions.items():
             # may need to change based on BGP config
             default = bgp['instance']['default']['vrf']['default']
```

Comment:
Let's be safe and also do the same here  default = bgp['instance']['default']['vrf']['default']

this is just ready to blow up too

Reply:
or better to change to checking all VRFs instead of only `default`.

Reply:
Changed to check all the vrf's